app/services/rss_reader/assembler.rb

```diff
@@ -96,10 +96,10 @@ def parse_and_translate_tweet!(html_doc)
         bq_with_p = bq.css("p")
         next if bq_with_p.empty?
 
-        second_content = bq_with_p.css("p")[1].css("a")[0].attributes["href"].value
```

Comment:
I'm not sure if the structure of this markup changed on Medium, but there's no more `p` tag inside `bq_with_p` only the `a` tag.